src/utils/customMessages.js

```diff
@@ -61,7 +61,10 @@ const messages = {
   invalidRole: 'The role must be Super Administrator, Travel Administrator, Travel Team Member, Manager or Requester',
   existingRole: 'That is the existing role for this user',
   superUser: 'You can not change the role for the super user',
-  noTwoSuperUser: 'You can not'
+  noTwoSuperUser: 'You can not',
```

Comment:
could you describe this message more

Reply:
As that message is not used, I was the one who created it and forgot to delete it he can delete it

Reply:
thank you @Ugizwenayo-Divine